go/src/client/lc0_main.go

```diff
@@ -0,0 +1,499 @@
+// A new client to work with the lc0 binary.
+//
+//
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"client/http"
+
+	"github.com/Tilps/chess"
+)
+
+var hostname = flag.String("hostname", "http://api.lczero.org", "Address of the server")
+var user = flag.String("user", "", "Username")
+var password = flag.String("password", "", "Password")
+var gpu = flag.Int("gpu", -1, "ID of the OpenCL device to use (-1 for default, or no GPU)")
+var debug = flag.Bool("debug", false, "Enable debug mode to see verbose output and save logs")
+
+type Settings struct {
+	User string
+	Pass string
+}
+
+//var lc0_args = []string{"--temperature=1", "--noise", "--no-smart-pruning", "--minibatch-size=1"}
```

Comment:
these args make lc0 much slower than lczero fyi.

Reply:
I think it's better to keep it fast and increase playouts so we know that no strength is loss. Maybe get a test that is +35 +-35 elo.

Reply:
I've changed default values for many parameters in selfplay mode, no need to set them up.
Other parameters can be taken from server. (There is a testing server now which is going to be used exclusively by new `lc0`, so we can setup parameters there).